### PR TITLE
feat: All tables show a loading indicator

### PR DESCRIPTION
**Pull request by xplato**
Adds the `LoadingToast` to all tables (except for the `SupportedDeviceTable`).

### CODE REVIEW

File: src/lib/seam/components/AccessCodeTable/AccessCodeTable.tsx [hunk 1]
```diff
@@ -178,6 +179,10 @@ export function AccessCodeTable({
     )
   }
 
+  if (isError) {
```
Comment (xplato) [inline src/lib/seam/components/AccessCodeTable/AccessCodeTable.tsx:182]: @razor-x was there a reason this wasn't included before like in the `DeviceTable`?

File: src/lib/seam/components/AccessCodeTable/AccessCodeTable.tsx [hunk 2]
```diff
@@ -200,6 +205,13 @@ export function AccessCodeTable({
             </IconButton>
           )}
         </div>
+        <div className='seam-table-header-loading-wrap'>
+          <LoadingToast
+            isLoading={isLoading}
+            label='Loading access codes'
```
Comment (razor-x) [inline src/lib/seam/components/AccessCodeTable/AccessCodeTable.tsx:211]: t.loading

**Review by razor-x — COMMENTED** [review]
Can we add to the other tables as well and release as feat: All tables show a loading indicator

**Comment by xplato** [comment]
@razor-x I added this to `ClimateSettingScheduleTable`, but for the `SupportedDeviceTable`, we might want @miken-seam to take a look. It has a loading state already, and with the animation it is a bit awkward. Could we hold off on this one for the time being and open an issue to revisit it right after?

**Comment by razor-x** [comment]
Just at lunch now, but testing this in mobile in the example app I'm seeing some odd behavior. First the location is off center.

Second, should we suppress the toast when data is already loaded? Meaning if I navigate around I don't think I should see the loading toast on every render (the react-query API should be able to help with this)

**Comment by xplato** [comment]
@razor-x Centered.

When you look at this on mobile, what size of device is it? Are any of these components mobilized? (Unless you just you're on-the-go, not on a mobile phone or whatever).

In regard to the rendering, are you just thinking `isLoading={isLoading && !isRefetching}`?

**Comment by razor-x** [comment]
Looks like centering is ok now.

And yes, not showing the loader on refetch sounds reasonable

**Comment by razor-x** [comment]
I feel like `isRefetching` should have fixed this, but testing here I still see the loader on every transition

https://react-hoay4cwu1-seamapi.vercel.app/examples/basic/


https://github.com/seamapi/react/assets/721372/2fcacaa5-ea03-4174-9ff9-60437a7f1179

**Comment by xplato** [comment]
> I feel like `isRefetching` should have fixed this, but testing here I still see the loader on every transition

@razor-x is this not because every transition is removing the entire component from the DOM? If the component is completely de-rendered, wouldn't we expect a fresh reload when it renders again?

**Comment by xplato** [comment]
@razor-x Ok, achieved with an additional timeout to remove the component from the DOM after the initial load.

**Review by razor-x — APPROVED** [review]

**Pull request merged**